ember-cli-build.js

```diff
@@ -273,19 +273,31 @@ module.exports = function (defaults) {
 
   replacementPattern = replacementPattern.concat(emberInspectorVersionPattern);
 
+  const firefoxBackgroundServiceReplacement = [
```

Comment:
What is this for? Firefox _should_ support manifest v3, right?

Reply:
It doesn't... Not all of it

Reply:
https://github.com/mozilla/web-ext/issues/2532